docs/content/en/docs/getting-started/site-engineer/_index.md

```diff
@@ -0,0 +1,232 @@
+    Release "k8ssandra" has been upgraded. Happy Helming!
+    NAME: k8ssandra
+    LAST DEPLOYED: Fri Feb 19 12:46:49 2021
+    NAMESPACE: default
+    STATUS: deployed
+    REVISION: 2
+    ```
+
+1. Verify that the Traefik pod is `Running`:
+
+    ```bash
+    kubectl get po -A | grep "traefik"
+    traefik       traefik-55996cbb6-v6s9p                1/1     Running     0          13m
+    ```
+
+Now you're ready to access the full suite of K8ssandra supporting applications.
+
+{{% alert title="Tip" color="success" %}}
+If you've followed the configuration instructions in this quick start, the URLs in the sections below should take you directly to the associated utilities.
+{{% /alert %}}
+
+## Access K8ssandra utilities
+
+K8ssandra includes the following bundled and customized utilities:
+
+### Cassandra Reaper
+
+[Cassandra Reaper](<"http://repair.127.0.0.1.nip.io:8080/webui">) provides an easy interface for managing K8ssandra cluster repairs. For details, see the [Cassandra Reaper](http://cassandra-reaper.io/) web site.
+
+### Prometheus
+
+[Prometheus](http://127.0.0.1.nip.io:8080/prometheus/) is a standard metrics collection and alerting tool. For more information, see the [Prometheus](https://prometheus.io/) web site.
+
+### Grafana
+
+[Grafana](http://127.0.0.1.nip.io:8080/grafana/login) provides a set of pre-configured dashboards displaying important K8ssandra metrics. For more information see the [Grafana](https://grafana.com/) web site.
+
+{{% alert title="Tip" color="success" %}}
+If you've followed the configuration instructions in this quick start, use `admin` for the username and `admin123` for the password.
+{{% /alert %}}
+
+### Medusa backup and restore
+
+K8ssandra provides a complete backup and restore solution using [Medusa](https://github.com/thelastpickle/cassandra-medusa). For detailed configuration and usage instructions, see [Backup and restore Cassandra]({{< relref "docs/topics/restore-a-backup" >}}).
+
+## Stopping and starting Cassandra {#cassandra-operations}
+
+Before shutting down your Kubernetes cluster, you'll want to make sure you cleanly shut down your Cassandra datacenters. You can do that using the `kubectl patch` command and setting the `spec:stopped` property to either `true` (stopped) or `false` (running).
+
+### Shut down Cassandra
+
+To shut down a Cassandra datacenter:
+
+```bash
+kubectl patch cassdc <datacenter-name> --type merge -p '{"spec":{"stopped":true}}'
+```
+
+Example:
+
+```bash
+kubectl patch cassdc dc1 --type merge -p '{"spec":{"stopped":true}}'
+cassandradatacenter.cassandra.datastax.com/dc1 patched
+```
+
+### Start up Cassandra
+
+To start up a Cassandra datacenter
+
+```bash
+kubectl patch cassdc <datacenter-name> --type merge -p '{"spec":{"stopped":false}}'
+```
+
+Example:
+
+```bash
+kubectl patch cassdc dc1 --type merge -p '{"spec":{"stopped":false}}'
+cassandradatacenter.cassandra.datastax.com/dc1 patched
+```
+
+## Nodetool fragment
```

Comment:
This seems unfinished.

Reply:
It's a to do. This whole section will be rewritten.